src/App.vue

```diff
@@ -149,11 +162,55 @@ export default {
         }
       },
       immediate: true
+    },
+    updating: {
+      handler: function(isUpdating, wasUpdating) {
+        window.clearInterval(this.updateStatusInterval);
+        // if updating, check loading status every two seconds
+        if (isUpdating) {
+          this.updateStatusInterval = window.setInterval(() => {
+            this.$store.dispatch("system/getUpdateStatus");
+          }, 2 * 1000);
+        } else {
+          //else check every minute
+          this.updateStatusInterval = window.setInterval(() => {
+            this.$store.dispatch("system/getUpdateStatus");
+          }, 60 * 1000);
+
+          // if it just finished updating, then show success/failure toast
+          if (wasUpdating) {
+            if (this.updateStatus.state === "success") {
+              this.$bvToast.toast(this.updateStatus.description, {
+                title: "Update successful",
+                autoHideDelay: 2000,
+                variant: "success",
+                solid: true,
+                toaster: "b-toaster-bottom-right"
+              });
+            } else if (this.updateStatus.state === "failed") {
+              this.$bvToast.toast(this.updateStatus.description, {
```

Comment:
Maybe dedupe this into a single call to toast() but just set the options in the if branches.

Reply:
Yeahhhh, cleaner.